packages/internal/src/build/api.ts

```diff
@@ -130,46 +129,14 @@ export const prebuildApiFiles = (srcFiles: string[]) => {
   })
 }
 
-// TODO: This can be shared between the api and web sides, but web
-// needs to determine plugins on a per-file basis for web side.
-export const prebuildFile = (
-  srcPath: string,
-  // we need to know dstPath as well
-  // so we can generate an inline, relative sourcemap
-  dstPath: string,
-  plugins: TransformOptions['plugins']
-) => {
-  const code = fs.readFileSync(srcPath, 'utf-8')
-
-  // @NOTE
-  // Even though we specify the config file, babel will still search for .babelrc
-  // and merge them because we have specified the filename property, unless babelrc = false
-  const result = transform(code, {
-    cwd: getPaths().api.base,
-    babelrc: false,
-    filename: srcPath,
-    configFile: getApiSideBabelConfigPath(),
-    // we set the sourceFile (for the sourcemap) as a correct, relative path
-    // this is why this function (prebuildFile) must know about the dstPath
-    sourceFileName: path.relative(path.dirname(dstPath), srcPath),
-    // we need inline sourcemaps at this level
-    // because this file will eventually be fed to esbuild
-    // when esbuild finds an inline sourcemap, it tries to "combine" it
-    // so the final sourcemap (the one that esbuild generates) combines both mappings
-    sourceMaps: 'inline',
-    plugins,
-  })
-  return result
-}
-
 export const transpileApi = (files: string[], options = {}) => {
   const rwjsPaths = getPaths()
 
-  return buildSync({
+  return esbuild.buildSync({
     absWorkingDir: rwjsPaths.api.base,
     entryPoints: files,
     platform: 'node',
-    target: 'node12.21', // AWS Lambdas support NodeJS 12.x and 14.x, but what does Netlify Target?
+    target: 'node12', // Netlify defaults NodeJS 12: https://answers.netlify.com/t/aws-lambda-now-supports-node-js-14/31789/3
```

Comment:
Was this the main reason we were hitting that problem with netlify deploys?

Reply:
It's possible that v8 in NodeJS12 was upgraded and there was some syntax that was not being transpiled properly.